packages/shared/src/components/post/PostItemCard.tsx

```diff
@@ -56,7 +57,7 @@ export default function PostItemCard({
         )}
       >
         <Image
-          src={post.image}
+          src={article.image}
```

Comment:
This one is also a bit confusing to read what it actually is

Reply:
When we say, we usually mean the actual post. And `article` is both true for `post` and `sharedPost` as the value can be either of the two.